meetingRooms/src/main/java/com/meetingRooms/dao/MemberScheduleDao.java

```diff
@@ -58,23 +57,15 @@ public MemberScheduleDao() {
 				m1.setOrganizedBy(rs.getString(3));
 				m1.setMeetingDate(rs.getDate(4));
 				m1.setStartTime((rs.getTime(5)));
-				m1.setDuration(Integer.parseInt(rs.getString(6)));
-				m1.setMeetingType(rs.getString(7));
+				m1.setMeetingRoom(rs.getString(7));
```

Comment:
Please also set the end time.

Reply:
i did.can u pull this, so that i can make a new pull request for these changes?